tests/util_test.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import jax
+import jax.numpy as np
+from jax import api, lax
+from jax import linear_util as lu
+from jax import test_util as jtu
+from jax import tree_util
+
+from jax.config import config
+config.parse_flags_with_absl()
+FLAGS = config.FLAGS
+
+class UtilTest(jtu.JaxTestCase):
+
+  def test_wrapped_fun_transforms(self):
+    """Test a combination of transforms."""
+
+    def f(*args, factor=2):
```

Comment:
It's 2020 so we could just drop py2 compatibility, but to make this work in py2 I think we need something like

```python
def f(*args, **kwargs):
  factor = kwargs.pop('factor', 2)
  assert not kwargs
  # ...
```